src/nft/pages/collection/index.tsx

```diff
@@ -1,5 +1,35 @@
-export const Collection = () => {
-  return <div>Collection Page</div>
+import { useWindowSize } from 'hooks/useWindowSize'
+import { GenieCollection } from 'nft/types'
+import { useEffect, useMemo } from 'react'
+import { useQuery } from 'react-query'
+import { useLocation, useParams } from 'react-router-dom'
+
+import { useIsMobile } from '../../hooks/useIsMobile'
+import { CollectionStatsFetcher } from '../../queries'
+import { CollectionDesktop } from './CollectionDesktop'
+import { CollectionMobile } from './CollectionMobile'
+
+const Collection = () => {
+  const { contractAddress } = useParams()
+
+  const isMobile = useIsMobile((state) => state.isMobile)
+  const setMobileWidth = useIsMobile((state) => state.setMobileWidth)
+  const location = useLocation()
+  const isActivityToggled = useMemo(() => location.pathname.includes('/activity'), [location])
```

Comment:
It's not obvious from the code what this is for. Can you either move this into the desktop and mobile components (so it is adjacent to the code for toggling activity), or define the callback to toggle activity in this file?

Otherwise, by splitting it, it's harder to understand - to borrow React terminology: it's like a semi-controlled component.

Reply:
honestly activity is out of the scope of this pr so I am just going to remove it until I add in the activity feature